bofire/data_models/features/continuous.py

```diff
@@ -20,7 +21,10 @@ class ContinuousInput(NumericalInput):
     type: Literal["ContinuousInput"] = "ContinuousInput"
     order_id: ClassVar[int] = 1
 
-    bounds: Tuple[float, float]
+    bounds: Tuple[float, float] = Field(description="mama")
```

Comment:
Oh, I was just testing the use of the description, and forget to remove it. I will remove it and also create an issue regarding docstrings and data model description fields, so that we can define a standard.